test/unit/helpers/clearStore.ts

```diff
@@ -1,9 +1,9 @@
 import store from '../../../src/store'
```

Comment:
Lets revert the changes to this file? Think we have a much better solution in master branch.